expression/builtin_time.go

```diff
@@ -5559,6 +5559,9 @@ func (b *builtinAddStringAndStringSig) evalString(row chunk.Row) (result string,
 	if isNull || err != nil {
 		return "", isNull, err
 	}
+	if !isDuration(arg1Str) {
+		return "", true, nil
+	}
 	sc := b.ctx.GetSessionVars().StmtCtx
 	arg1, err = types.ParseDuration(sc, arg1Str, getFsp4TimeAddSub(arg1Str))
```

Comment:
I'm more expect here throw a `ErrTruncatedWrongVal` error, IMO, it should be (**in fact no**). @wshwsh12 what do you think? more details I was said in Slack. please check it out.

Reply:
I don't think so. MySQL also doesn't throw a error/warning.

Reply:
It's make sense. Should we remove those two test [cases](https://github.com/hidehalo/tidb/blob/issue-19150/expression/builtin_time_test.go#L997)(below)? Or we should throw a error/warning inside `func (b *builtinAddStringAndStringSig) evalString(row chunk.Row)`?

Reply:
@wshwsh12 PTAL PR https://github.com/pingcap/tidb/pull/11262, I really feel confuse, would you like to give me some advices? thx!